doc/api/fs.md

```diff
@@ -741,6 +741,40 @@ Returns an object containing commonly used constants for file system
 operations. The specific constants currently defined are described in
 [FS Constants][].
 
+## fs.copyFile(src, dest[, flags], callback)
+<!-- YAML
+added: REPLACEME
+-->
+
+* `src` {string|Buffer|URL} source filename to copy
+* `dest` {string|Buffer|URL} destination filename of the copy operation
+* `flags` {number} modifiers for copy operation. **Default:** `0`
```

Comment:
Are we *certain* this should be the default? I don't want to see bug reports later about how Node.js made it so that their files were all screwed up... and there is a non-zero chance of abusing this.

Reply:
Yes. `cp` defaults to overwriting, as does our own `fs.rename()`.

Reply:
Yeah, I'm aware, just makes me rather cringy. Guess I'll grit my teeth and bear it.